packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/Convert.java

```diff
@@ -464,36 +439,6 @@ static LatLng latLngFromPigeon(Messages.PlatformLatLng latLng) {
     return new LatLng(latLng.getLatitude(), latLng.getLongitude());
   }
 
-  static Object clusterToJson(String clusterManagerId, Cluster<MarkerBuilder> cluster) {
```

Comment:
This is pure removal because the Pigeon version (just below) was added in a previous PR.